src/world/Management/Gossip/GossipDefines.hpp

```diff
@@ -23,13 +23,14 @@ struct GossipItem
 
 struct GossipQuestItem
 {
-    GossipQuestItem(uint8_t icon, uint32_t level = 0, uint32_t flags = 0) :
-        icon(icon), level(level), flags(flags) {}
+    GossipQuestItem(uint8_t icon, int32_t level = 0, uint32_t flags = 0, uint8_t repeatable = 0) :
+        icon(icon), level(level), flags(flags), repeatable(repeatable) {}
 
     uint8_t icon;
+    int32_t level;
     // used for clients > tbc
-    uint32_t level;
```

Comment:
What's the motivation for changing this field?

Reply:
@Evairfairy Questlevel in quest_properties is a signed int. Currently min_level is sent for quest level (as can be seen here https://github.com/AscEmu/AscEmu/pull/802/files#diff-b84aea28f707eedf5b724610266d6d5fL31) which is unsigned but that is inaccurate for the gossip packet.
Sending min_level in the packet would make a quest appear as "(low level)" in gossip menu too early.